test/e2e/pod_cleanup_test.go

```diff
@@ -21,7 +21,7 @@ type PodCleanupSuite struct {
 	fixtures.E2ESuite
 }
 
-const enoughTimeForPodCleanup = 3 * time.Second
+const enoughTimeForPodCleanup = 33 * time.Second
```

Comment:
Padding out tests as they will need a little more time to "wait" now as there is a default deletion delay.